### PR TITLE
Quora Card for websites.

### PR DESCRIPTION
**Pull request by arpitbbhayani**

### CODE REVIEW

File: files/quora-widget/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "quora-widget",
+  "repo": "arpitbbhyani/quora-widget",
+  "files": {
+    "include": ["quora-widget.js", "quora-widget.min.js"]
```
Comment (jimaek) [inline files/quora-widget/update.json]: I dont think `quora-widget.js` is needed in production. The min is enough

Comment (arpitbbhayani) [inline files/quora-widget/update.json]: @jimaek  I have removed the entry of `quora-widget.js`. Thanks for pointing that out.

File: files/quora-widget/update.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "quora-widget",
+  "repo": "arpitbbhyani/quora-widget",
```
Comment (jimaek) [inline files/quora-widget/update.json]: There is a typo here. its not correct

Comment (arpitbbhayani) [inline files/quora-widget/update.json]: Fixed tyops. Thanks.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**